WooCommerce/Classes/Model/MarkOrderAsReadUseCase.swift

```diff
@@ -18,15 +18,16 @@ struct MarkOrderAsReadUseCase {
         case unavailableNote
         case noNeedToMarkAsRead
     }
+
+#if canImport(Yosemite)
     /// Async method that marks the order note as read if it is the notification for the last order.
     /// We do it in a way that first we syncronize notification to get the remote `Note`
     /// and then we compare local `orderID` with the one from remote `Note`.
     /// If they match we mark it as read.
     /// Returns syncronized note if marking was successful and error if some error happened
-#if canImport(Yosemite)
```

Comment:
Moved the header doc within the `#if` branch that conditionally defines the method the header doc documents